### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by achang28**

### CODE REVIEW

File: js/KmPlot.js [hunk 1]
```diff
@@ -240,30 +280,12 @@ var KmPlot = React.createClass({
...
-		let dims = {};
-
-		_.each(this.size.ratios, (sectionRatios, sectionName) => {
-			dims[sectionName] = _.mapObject(sectionRatios, (ratio, param) => {
-				return ratio * viewDims[param];
-			});
-		});
-
-		return dims;
+	componentWillMount: function() {
```
Comment (acthp) [inline js/KmPlot.js]: cut this fn

File: js/Application.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ var Application = React.createClass({
 //			Perf.printWasted();
 //		}
 //	},
+
```
Comment (acthp) [inline js/Application.js]: drop whitespace changes in this file

File: js/KmPlot.js (hunk 1 above)
Comment (achang28) [inline js/KmPlot.js]: The operation within componentWillMount is meant to calculate the width for the 'definitions' section, as it is a function of the width the graph:  graph's width % + definitions width % = 100%

Comment (acthp) [inline js/KmPlot.js]: They should be constant in this.size, and so not part of the component lifecycle. Either hard-code it to 1 - width, or move some part of it outsize of the component, e.g.

```javascript
const graphWidth = 0.75, defsWidth = 1 - graphWidth;
```

or

```javascript
const graphRatio = {width: 0.75, height: 1};
const defsRatio = {width: 1 - graphRatio.width, height: 1};
```
then reference these vars in the ```size``` declaration.

But, really, hard-coding 0.25 in this case is not a bad option.

Comment (achang28) [inline js/KmPlot.js]: okay cool.  Will just specific 0.25.

**Comment by achang28** [comment]
This PR resolves https://github.com/acthp/ucsc-xena-client/issues/17

**Pull request merged**